account/recovery.go

```diff
@@ -17,6 +20,69 @@ var (
 	DefaultAccountKeyWindow uint32 = 500
 )
 
+// auctioneerData contains the auctioneers' information needed to recovery
+// the user's accounts without interacting with the server.
+//
+// Note: Each environment has its own values.
+type auctioneerData struct {
+	// Initial value for the batch key.
+	InitialBatchKey *btcec.PublicKey
+
+	// Auctioneer public key.
+	PubKey *btcec.PublicKey
+
+	// Block height for the first batch ever.
+	FirstBatchBlock uint32
+}
+
+// getAuctioneerData returns the auctioner data for a given environment.
+func getAuctioneerData(network string) (*auctioneerData, error) {
+	var auctioneerKey string
+	var batchKey string
+	var fstBlock uint32
+
+	switch network {
+	case "mainnet":
+		auctioneerKey = "034d1fd68b5cd92f694766aa8b3f90131613d1b731" +
```

Comment:
Where did you get that value from? I think it's actually `028e87bdd134238f8347f845d9ecc827b843d0d1e27cdcb46da704d916613f4fce` (also see https://github.com/guggero/chantools/blob/master/cmd/chantools/closepoolaccount.go#L37 which was successfully used to recover an account).

Reply:
updated, I thought I got it from your chantools but maybe I updated it for testing and forgot about it.